File: lab_2/src/ArrayGenerator.java
import java.util.Random;

public class ArrayGenerator {
    public static void generate(int[] array) {
        Random random = new Random();
        for (int i = 0; i < array.length; i++) {
            array[i] = random.nextInt(100000);
        }

        int negativeIndex = random.nextInt(array.length);
        array[negativeIndex] = -random.nextInt(1000);
        System.out.println("Negative number inserted at index: " + negativeIndex + " (value: " + array[negativeIndex] + ")");
    }
}

File: lab_2_C#/ArrayGenerator.cs
using System;

class ArrayGenerator
{
    public static void Generate(int[] array)
    {
        Random random = new Random();
        for (int i = 0; i < array.Length; i++)
        {
            array[i] = random.Next(100000);
        }

        int negativeIndex = random.Next(array.Length);
        array[negativeIndex] = -random.Next(1000);

        Console.WriteLine($"Negative number inserted at index: {negativeIndex} (value: {array[negativeIndex]})");
    }
}
